CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Snyk Security - Code and Open Source Dependencies Changelog
 
+## [1.2.25]
+
+### Feat
```

Comment:
We usually have "Added", "Fixed", "Changed" categories in changelog if you look below.